packages/react/.storybook/theme.ts

```diff
@@ -1,12 +1,11 @@
-// .storybook/YourTheme.js
-
 import { create } from "@storybook/theming/create";
+//@ts-ignore
```

Comment:
Why ts-ignore? 

If it due to an issue in the brand assets package, lets create another issue to fix it and add a note here about why the ignore was added.